src/components/FiltersList/presenter.jsx

```diff
@@ -54,32 +58,33 @@ function FilterFormSelect(props) {
   } = props;
 
   return <Select
-    isMulti={isMulti}
-    mods={mods}
-    placeholder={placeholder}
-    options={options}
-    value={input.value}
-    expanded={meta.active}
-    disabled={meta.submitting || disabled}
-    onFocus={() => input.onFocus()}
-    onBlur={() => input.onBlur()}
-    onChange={(val) => {
-      let newValue = val;
-      if (isMulti) {
-        const isAnyOptionSelected = val.includes(anyOptionValue);
-        newValue = isAnyOptionSelected ? [] : val;
-      } else {
-        // whether 'Any' option selected
-        newValue = val === anyOptionValue ? null : val;
-      }
-      input.onChange(newValue);
-    }}
+      isMulti={isMulti}
```

Comment:
Remove all dummy-formatting related diff

Reply:
I've applied .eslintrc rules stored in project but there are some diffs still